ci-operator/step-registry/openshift/e2e/cert-rotation/prepare-for-shutdown/openshift-e2e-cert-rotation-prepare-for-shutdown-commands.sh

```diff
@@ -53,20 +53,22 @@ cat << 'EOZ' > /tmp/approve-csrs-with-timeout.sh
   export KUBECONFIG=/etc/kubernetes/static-pod-resources/kube-apiserver-certs/secrets/node-kubeconfigs/localhost-recovery.kubeconfig
   echo -n "Approving CSRs"
   attempts=0
-  max_attempts=10
+  max_attempts=5
   while true; do
     echo -n '.'
-    mapfile -d ' ' -t csrs < <(oc get csr | grep Pending | cut -f1 -d" ")
+    mapfile -d ' ' -t csrs < <(oc get csr --field-selector=spec.signerName='kubernetes.io/kube-apiserver-client-kubelet' --no-headers | grep Pending | cut -f1 -d" ")
```

Comment:
Why do we selectively approve？ not all pednding csrs?

Reply:
If we don't care the csr name, give it a try with following,
```
if oc get csr | grep Pending ;then oc get csr | grep Pending | awk '{print $1}' | xargs oc adm certificate approve;fi
```

Reply:
Some CSRs are created by openshift-monitoring and meant to be approved by openshift-apiserver. This change will ensure that during time skew openshift-apiserver still functions (or picks up existing CSRs).